test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -181,8 +235,10 @@ func TestEtcdStoragePath(t *testing.T) {
 	}
 }
 
+var debug = false
+
 func dumpEtcdKVOnFailure(t *testing.T, kvClient clientv3.KV) {
-	if t.Failed() {
+	if t.Failed() && debug {
```

Comment:
Is debug setting to true anywhere?

Reply:
no. this output is excessively verbose and is not relevant for most test failures... keeping this here is fine, but requiring someone to flip the debug var to true when debugging a failure locally

Reply:
Thanks for the explanation.